package demo;

message msg
{
    required int32 MsgType=1;
    required string MsgInfo=2;
    required string MsgFrom=3;
    optional string opt=4; //can select 
}


